packages/cf-component-icon/src/Icon.js

```diff
@@ -1,33 +1,245 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-class Icon extends React.Component {
-  render() {
-    let className = `cf-icon cf-icon--${this.props.type}`;
+import { createComponent } from '../../cf-style-container';
+
+const Wrapper = createComponent(({
+  color = '#333333',
+  fontSize = 'initial',
+  border = false,
+  spin = false,
+  type,
+  theme
```

Comment:
I was hoping to get rid of those with this change :P seemed to be just changing the color and you should be able to do it yourself with theming.